packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -476,11 +507,40 @@ const ReviewVotingPage = ({classes}: {
                 </div>
               </LWTooltip>}
             </>}
-            <LWTooltip title="Sorts the list of posts by vote strength">
-              <Button onClick={reSortPosts}>
-                Re-Sort <CachedIcon className={classes.menuIcon} />
-              </Button>
+            <LWTooltip title={`Sorted by ${sortReversed ? "Descending" : "Ascending"}`}>
+              <div onClick={() => { 
+                setSortReversed(!sortReversed); 
+                reSortPosts(sortPosts, !sortReversed);
```

Comment:
I don't think you need to call `reSortPosts()` here or below, since changing the sort criteria will already trigger `reSortPosts()` via the `useEffect`. This causes it to run twice.